packages/shared/utils.sh

```diff
@@ -0,0 +1,3 @@
+dir() {
```

Comment:
Where is this being used?

Reply:
Leftover from learning the test workflow-- I'll remove it.